repository/gogit_testing.go

```diff
@@ -1,21 +1,28 @@
 package repository
 
 import (
-	"io/ioutil"
 	"log"
+	"path/filepath"
+	"strings"
+	"testing"
 
 	"github.com/99designs/keyring"
 )
 
+type TestingT interface {
```

Comment:
Hmm, why defining this interface? If you don't want to bind to `testing.T`, maybe use `testing.TB`?